docs/Oracle Jobs/Job Types/keeper.md

```diff
@@ -11,22 +11,58 @@ Keeper jobs occasionally poll a smart contract method that expresses whether som
 - Liquidations
 - Rebalancing portfolios
 - Rebase token supply adjustments
+- Auto-compounding
+- Limit orders
 
 **Spec format**
 
 ```jpv2
 type            = "keeper"
-schemaVersion   = 1
+schemaVersion   = 3
+evmChainID      = "1"
 name            = "example keeper spec"
-contractAddress = "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba"
+contractAddress = "0x7b3EC232b08BD7b4b3305BE0C044D907B2DF960B"
 fromAddress     = "0xa8037A20989AFcBC51798de9762b351D63ff462e"
 externalJobID   = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F04"
+observationSource = """
```

Comment:
I tested this on a 1.3.0 node and something is wrong with the observationSource. I keep getting a `invalid observation source provided` error when I try to create the job even though the task list looks correct. I made a few changes to improve syntax highlighting but couldn't figure out what was preventing this job from being created.